api-reference/10 UI Widgets/dxTreeList/1 Configuration/onEditorPreparing.md

```diff
@@ -53,13 +52,13 @@ The [properties](/api-reference/10%20UI%20Widgets/dxTreeList/6%20Row '/Documenta
 Indicates whether the editor uses right-to-left representation.
 
 ##### field(e.setValue(newValue, newText)): any
-A method you should call to change the cell value and, optionally, the displayed value after the editor's value is changed.
+Changes the cell value and, optionally, the displayed value after the editor's value is changed.
```

Comment:
Obviously, all the comments I left for the DataGrid's `onEditorPreparing` also apply to the TreeList's option. We better modify this article so that function parameters inherit their descriptions from the grid's version.